lib/suspicious_login/model.rb

```diff
@@ -58,7 +58,7 @@ def suspicious?(request = {})
       end
 
       def send_suspicious_login_instructions(request = {})
-        send_devise_notification(:suspicious_login_instructions, nil, {})
+        send_devise_notification(:suspicious_login_instructions, nil, {}) unless Devise.skip_suspicious_login_email == true
```

Comment:
would it be better to not call send_suspicious_login_instructions unless skip_suspicious_login_email?

Reply:
`send_suspicious_login_instructions` can be overwritten in the hub/auth to execute custom logic. Ie. send a rabbit message instead of an email